src/Aspire.Hosting.Azure.WebPubSub/AzureWebPubSubExtensions.cs

```diff
@@ -0,0 +1,75 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Azure;
+
+using Azure.Provisioning;
+
+using Azure.Provisioning.WebPubSub;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding the Azure Web PubSub resources to the application model.
+/// </summary>
+public static class AzureWebPubSubExtensions
+{
+    /// <summary>
+    /// Adds an Azure Web PubSub resource to the application model.
+    /// Change sku: WithParameter("sku", "Standard_S1")
+    /// Change capacity: WithParameter("capacity", 2)
+    /// </summary>
+    /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
+    /// <param name="name">The name of the resource. This name will be used as the connection string name when referenced in a dependency.</param>
+    /// <returns>A reference to the <see cref="IResourceBuilder{T}"/>.</returns>
+    public static IResourceBuilder<AzureWebPubSubResource> AddAzureWebPubSub(this IDistributedApplicationBuilder builder, string name)
```

Comment:
Could we get some examples added here e.g.:

```csharp
/// <example>
/// Brief summary of example
/// <code>
/// ....
/// </code>
```

Reply:
... and on the other extension methods.

Reply:
Is <example> a supported tag in triple slash comment?